manifests/kots-config.yaml

```diff
@@ -903,6 +891,27 @@ spec:
           required: true
           default: "3"
 
+        ## Custom Encryption Keys
+        - name: enableCustomEncryptionKeys
```

Comment:
But we said that this option is going to be hidden and only enabled to some clients by demand, right? :thinking:

Reply:
Oh yes 🙇🏼‍♂️ Fixed!